wnfs/src/private/file.rs

```diff
@@ -668,6 +787,30 @@ mod tests {
             content[2 * MAX_BLOCK_CONTENT_SIZE..4 * MAX_BLOCK_CONTENT_SIZE]
         );
     }
+
+    #[async_std::test]
+    async fn can_construct_file_from_stream() {
+        let disk_file = File::open("./src/private/directory.rs").await.unwrap();
```

Comment:
Fixture? Source file likely to change.

Reply:
Yeah. You're right. I should just find some decent fixture.